src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import { User } from '../../types';
+
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({
+  user: {
```

Comment:
I'm not sure that it's a good idea to do it here. I think you should do destructuring in the body of the component.

const { name, email } = user;

Reply:
> I'm not sure that it's a good idea to do it here. I think you should do destructuring in the body of the component.
> 
> const { name, email } = user;

Good point